LanguageFeatures/Enhanced-Enum/implementing_enum_A03_t04.dart

```diff
@@ -18,15 +18,13 @@
 /// Such an instance `values` declaration is either useless or wrong, so we
 /// disallow it entirely.
 ///
-/// @description Check that it's a compile-time error if a class has Enum as a
+/// @description Check that it's a compile-time error if a class has `Enum` as a
 /// superinterface and the interface of the declarations contains an instance
-/// member with the name values
+/// member with the name `values`.
```

Comment:
Thanks! It's good for the overall readability if these quotes are added whenever the library is being updated anyway.